README.md

```diff
@@ -11,15 +11,28 @@
 
 # Cacator
 
-Cacator (с лат. — "засранец"). Маячок для выявления несанкционированных копий нашего ПО. Работает через DNS covert channels.
+[RU](./README_RU.md)
 
-Проект является глубокой переработкой [WEASEL](https://github.com/facebookarchive/WEASEL).
+Cacator, derived from Latin, meaning "defecator," sounds like "pain in the a**" in Russian.
```

Comment:
this can break markdown... so let's make it **safe as \*\*\*\*** :trollface: (by adding escapes)

```suggestion
Cacator, derived from Latin, meaning "defecator," sounds like "pain in the a\*\*" in Russian.
```